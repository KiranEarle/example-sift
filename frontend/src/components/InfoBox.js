import React from 'react'

export const InfoBox = () => {
  return (
    <section className='info-box'>
      <p>
        Here are a list of your emails, click "more details" to get more information.
      </p>
      <p>
        Remember if the "from" &amp; "Return Path Domain" do not have the same domain
        name, this mail cannot be trusted.
      </p>
    </section>
  )
}
