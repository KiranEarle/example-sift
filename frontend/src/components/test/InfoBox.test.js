import React from 'react'
import { InfoBox } from '../InfoBox'

it('render without blowing up', () => {

  const component = render(<InfoBox />);
  expect(component).toMatchSnapshot();
});