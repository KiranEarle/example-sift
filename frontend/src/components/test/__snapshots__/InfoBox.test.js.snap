// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render without blowing up 1`] = `
<section
  class="info-box"
>
  <p>
    Here are a list of your emails, click "more details" to get more information.
  </p>
  <p>
    Remember if the "from" & "Return Path Domain" do not have the same domain name, this mail cannot be trusted.
  </p>
</section>
`;
